updates/location_asset_mappings/v9.json

```diff
@@ -4649,6 +4639,26 @@
       {
         "location": "bitfinex",
         "location_symbol": "MNXT"
+      },
+      {
+        "location": "kraken",
+        "location_symbol": "BNB"
+      },
+      {
+        "location": "kraken",
+        "location_symbol": "RNDR"
+      },
+      {
+        "location": "kraken",
+        "location_symbol": "DEGEN"
+      },
```

Comment:
you deleted this one from this file above. Does it need a delete statement?

Reply:
> Does it need a delete statement?

oh it does not. removed.